lib/commands/help.js

```diff
@@ -34,6 +44,35 @@ function activate(eName, message){
     return;
   }
 
-  return message.respond(`My commands are as follows:
-${JSON.stringify(help, null, 4)}`);
+  //handle general help
+  if(!msg[1]) {
+    let resp = 'Here are my commands:\n';
+    resp+= '```';
+    help.forEach((h)=>{
+      resp += `${_.padEnd(h.cmd, 12)}   ${h.description}\n`
+    });
+    resp+='```';
+    return message.respond(resp);
+  }
+
+  //handle help for a given command
+  let searchCmd = msg[1];
+  let resp = '```\n';
+  for(let i = 0; i < help.length; i++){
+    let cmd = help[i];
+    if(cmd.cmd === searchCmd || cmd.aliases.includes(searchCmd)){
+      //found it
+      resp += `${_.padEnd(cmd.cmd, 12)}\t${cmd.usage || ''}\n`;
```

Comment:
This line isn't very readable. I understand it after reading it a few times but there's potential to do it in multiple lines or declare variables for some of the logic outside of the template literal in my opinion.

Again, not a big enough deal to stop an approval. Just a thought.